### ISSUE TITLE
Can't draw polygons when using setOptIn(true)

### ISSUE THREAD

**Issue by Patabugen** [geoman-io/leaflet-geoman#795]
Hey,

I've come across a bug whereby I cannot draw polygons while also using `L.PM.setOptIn(true);`

JS Fiddle: (It's your template with the events removed and just that one line added at the bottom).

https://jsfiddle.net/Patabugen/4mndzhj7/5/

Open the browser console and try and draw a polygon - the error comes from the `debugIndicatorLines[1]` being undefined as part of the `_syncHintMarker` and is possibly related to snapping?

![image](https://user-images.githubusercontent.com/196795/108211139-437da080-7124-11eb-9096-e925dffb8a81.png)

This seem to pass the error on a step further up the chain in case that's of any use:
```
map.on('pm:drawstart', function (e) {
  e.workingLayer.options.pmIgnore = false;
  L.PM.reInitLayer(e.workingLayer);
});
```

**Comment by Falke-Design** [comment]
Thx for reporting. This is already fixed in the develop branch through #764

**Comment by Patabugen** [comment]
Ah, amazing! Thanks!
